### PR TITLE
hardinfo.md: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/hardinfo.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# hardinfo
+
+> Show hardware information in GUI window.
+
+- Start hardinfo:
```
Comment (sbrl) [inline pages/linux/hardinfo.md]: Perhaps we could include another example or two? `hardinfo --help` outputs this for me: https://starbeamrainbowlabs.com/bin/ms05

Comment (jsonbruce) [reply pages/linux/hardinfo.md]: Yes, You're right.
Actually most of the time we use `hardinfo` as GUI tool, but it also can output hardware information to stdout.
I should add the other situation even if we rarely use it.

Comment (sbrl) [reply pages/linux/hardinfo.md]: Ah, ok. I guess the generation of a report might be useful in an automation scenario - e.g. where one has 100 machines and needs information on all their hardware for an audit or something.

Comment (jsonbruce) [reply pages/linux/hardinfo.md]: Yes, it's a good application situation.

**Pull request merged**